common/src/warp_runner/manager/commands/multipass_commands.rs

```diff
@@ -395,6 +400,23 @@ pub async fn handle_multipass_cmd(cmd: MultiPassCmd, warp: &mut super::super::Wa
                     rsp.send(Err(e))
                 }
             };
+        }
+        MultiPassCmd::GetIdentity { did, rsp } => {
+            let pic = warp.multipass.identity_picture(&did).await;
+            let r = warp.multipass.get_identity(Identifier::DID(did)).await; //(dids, &mut warp.multipass).await;
+            let r = match r {
+                Ok(ids) => {
+                    if ids.is_empty() {
+                        Err(Error::IdentityExist)
```

Comment:
if the vector is empty, then the identity wouldnt exist

```suggestion
                        Err(Error::IdentityDoesntExist)
```